added schedule and schedule shift documentation (#36)
TamakiControl/tamaki-mes-docs: docs/appendix/data-model/production-order-model/production-order.md

References the `ProductionOrderCustomer` entity associated with the production order, providing us information on the customer.
See [production_order_customers](../production-order-model/production-order-customer) for details.

### `schedule_shift_record_id`

> References the `ScheduleShiftRecord` entity that defines the shift during which this production order is scheduled. This relationship connects the production order to its specific shift instance, allowing for better tracking of production orders within the context of scheduled shifts.

Reviewer: ```suggestion
References the `ScheduleShiftRecord` entity that was active at the location the production order ran. This relationship connects the production order to its specific shift instance, allowing for better tracking of production orders within the context of scheduled shifts.
```
